data/core/images/units/elves-wood/prowler.cfg

```diff
@@ -0,0 +1,131 @@
+#textdomain wesnoth-celmin-aoa
+
+[unit_type]
```

Comment:
Why the heck is CFG file inside images???

Reply:
Huh? I did accidentally do this but I thought I'd already deleted it. Oops!